core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -194,7 +194,8 @@ private[spark] object JsonProtocol {
     ("App Name" -> applicationStart.appName) ~
     ("App ID" -> applicationStart.appId.map(JString(_)).getOrElse(JNothing)) ~
     ("Timestamp" -> applicationStart.time) ~
-    ("User" -> applicationStart.sparkUser)
+    ("User" -> applicationStart.sparkUser) ~
+    ("appAttemptId" -> applicationStart.appAttemptId.map(JString(_)).getOrElse(JNothing))
```

Comment:
You should add a test case here: 
https://github.com/apache/spark/blob/fbe7106d75c6a1624d10793fba6759703bc5c6e6/core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala#L275